manager/testdata/notebook/README.md

```diff
@@ -0,0 +1,347 @@
+
+Notice the `metadata` field above. It specifies the dataset geography and tags. These attributes can later be used in policies.
+
+For example, in the yaml above, the `geography` is set to `theshire`, you need make sure it is same with the region of your fybrik control plane, you can get the information with the below command:
+
+```shell
+kubectl get configmap cluster-metadata -n fybrik-system -o 'jsonpath={.data.Region}'
+```
+
+[Quick Start](../get-started/quickstart.md) installs a fybrik control plane with the region `theshire` by default. If you change it or the `geography` in the yaml above, a [copy module](https://github.com/fybrik/mover) will be required by the policies, but we do not install any copy module in the [Quick Start](../get-started/quickstart.md).
+
+## Define data access policies
+
+Define an [OpenPolicyAgent](https://www.openpolicyagent.org/) policy to redact the columns tagged as `PII` in datasets tagged with `finance` when reading data and to redact the columns tagged as `sensitive` in datasets tagged with `finance` when writing data. Below is the policy (written in [Rego](https://www.openpolicyagent.org/docs/latest/policy-language/#what-is-rego) language):
+
+```rego
+package dataapi.authz
+
+rule[{"action": {"name":"RedactAction", "columns": column_names}, "policy": description}] {
+  description := "Redact columns tagged as sensitive in datasets tagged with finance = true in write flow"
+  input.action.actionType == "write"
+  input.resource.metadata.tags.finance
+  column_names := [input.resource.metadata.columns[i].name | input.resource.metadata.columns[i].tags.sensitive]
+  count(column_names) > 0
+}
+
+rule[{"action": {"name":"RedactAction", "columns": column_names}, "policy": description}] {
+  description := "Redact columns tagged as PII in datasets tagged with finance = true in read flow"
+  input.action.actionType == "read"
+  input.resource.metadata.tags.finance
+  column_names := [input.resource.metadata.columns[i].name | input.resource.metadata.columns[i].tags.PII]
+  count(column_names) > 0
+}
+```
+
+In this sample only the policy above is applied. Copy the policy to a file named `sample-policy.rego` and then run:
+
+```bash
+kubectl -n fybrik-system create configmap sample-policy --from-file=sample-policy.rego
+kubectl -n fybrik-system label configmap sample-policy openpolicyagent.org/policy=rego
+while [[ $(kubectl get cm sample-policy -n fybrik-system -o 'jsonpath={.metadata.annotations.openpolicyagent\.org/policy-status}') != '{"status":"ok"}' ]]; do echo "waiting for policy to be applied" && sleep 5; done
+```
+
+You can similarly apply a directory holding multiple rego files.
+
+## Deploy a Jupyter notebook
+
+Refer to `Deploy a Jupyter notebook` section in [fybrik notebook sample](https://fybrik.io/v0.6/samples/notebook/).
+
```

Comment:
I would suggest adding somewhere an explanation of what is implemented in the notebook sample, probably at the very beginning.  Otherwise it's quite difficult to understand the reason for the steps being done.